components/raftstore/src/store/worker/read.rs

```diff
@@ -786,26 +933,27 @@ impl<'r, 'm> RequestInspector for Inspector<'r, 'm> {
 
 const METRICS_FLUSH_INTERVAL: u64 = 15_000; // 15s
 
+/// The reason for it to be pub is that it is shared with raftstore v2.
```

Comment:
I think @tonyxuqqi 's original suggestion is better, "#[RaftstoreCommon]" makes it easy to search. Ideally when the v2 works are all finished, we could gather and re-export these things in a separate namespace.

Reply:
And BTW, there seems to be a lot of shared structs/traits not annotated.